### ISSUE TITLE
Network protocol error, or freeze restarting

### ISSUE THREAD

**Issue by ghost** [ReplayMod/ReplayMod#1019]
So, if I try to open old realms/server replays that are pre 1.21.3, they don't open and display network protocol error, and If I open old single player recordings from pre 1.21.3 they open, but get stuck in a reloading loop whether I pause or play them.

[disconnect-2024-11-14_12.49.05-client.txt](https://github.com/user-attachments/files/17757648/disconnect-2024-11-14_12.49.05-client.txt)
I don't know if the disconnect shows other mods interfering or not, but I tried this with no mods but replaymod and fabric api and it still happened.

**Comment by mharbb** [comment]
> Same issue here. Seems to be with a specific replay file.

Yeah, if you really need to open old replay mod files, you can use replay mod 1.21.1 or below, but they should definitely look into instead of radio silence.
